### PR TITLE
specify partId mode

### PR DESCRIPTION
**Pull request by panda-sheep**
#### What type of PR is this?
- [ ] bug
- [x] feature
- [ ] enhancement

#### Checklist：
- [ ] Documentation affected （Please add the label if documentation needs to be modified.)
- [ ] Incompatible （If it is incompatible, please describe it and add corresponding label.）
- [ ] Need to cherry-pick （If need to cherry-pick to some branches, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

#### Release notes：
Please confirm whether to reflect in release notes and how to describe:
>                                                                 `

### CODE REVIEW

File: src/tools/db-upgrade/DbUpgrader.cpp [hunk 1]
```diff
@@ -82,10 +87,44 @@ Status UpgraderSpace::initSpace(const std::string& sId) {
...
+        partId = folly::to<PartitionID>(part);
+      } catch (const std::exception& ex) {
+        LOG(ERROR) << "Cannot convert part id " << part;
+        return Status::Error("Cannot convert part id %s", part.c_str());
+      }
+      partsAll.emplace_back(partId);
+    }
+
+    // to check in rocksdb
+    for (auto part : partpartsAlls) {
```
Comment (critical27) [inline src/tools/db-upgrade/DbUpgrader.cpp:115]: If the we can't find every part in `FLAGS_partIds` in local rocksdb , we should return error.

Comment (panda-sheep) [reply src/tools/db-upgrade/DbUpgrader.cpp]: Yeah, direct `LOG(FATAL)` and exit

File: src/tools/db-upgrade/DbUpgrader.cpp [hunk 2]
```diff
@@ -22,10 +24,15 @@ DEFINE_string(dst_db_path,
               "multi paths should be split by comma");
 DEFINE_string(upgrade_meta_server, "127.0.0.1:45500", "Meta servers' address.");
 DEFINE_uint32(write_batch_num, 100, "The size of the batch written to rocksdb");
-DEFINE_uint32(upgrade_version,
+DEFINE_uint32(raw_data_version,
```
Comment (darionyaphet) [inline src/tools/db-upgrade/DbUpgrader.cpp:27]: The name is still very difficult to understand

Comment (panda-sheep) [reply src/tools/db-upgrade/DbUpgrader.cpp:27]: Yeah? I feel more intuitive

**Comment by panda-sheep** [comment]
close this pr, will move to v2.6.1-nx branch.

**Pull request closed without merging**